### PR TITLE
Enable release for rh-trex

### PR DESCRIPTION
**Pull request by gurnben**
## Summary of Changes

This PR roughly mirrors the configuration used by redhat-acm into the rh-trex workspace to enable release to quay!

### CODE REVIEW

File: cluster/stone-prd-rh01/tenants/rh-trex-tenant/release-plan.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+apiVersion: appstudio.redhat.com/v1alpha1
+kind: ReleasePlan
+metadata:
+  labels:
+    release.appstudio.openshift.io/auto-release: 'true'
+    release.appstudio.openshift.io/standing-attribution: 'true'
+  name: rh-trex-release-to-quay
+spec:
+  application: rh-trex
+  target: rh-trex-tenant
```
Comment (ernesgonzalez33) [inline cluster/stone-prd-rh01/tenants/rh-trex-tenant/release-plan.yaml:11]: The target needs to be the managed namespace. In this case: `rh-managed-mng-s-2-tenant`

**Review by ernesgonzalez33 — CHANGES_REQUESTED** [review]

File: cluster/stone-prd-rh01/tenants/rh-trex-tenant/release-plan.yaml (hunk 1 above)
Comment (gurnben) [reply cluster/stone-prd-rh01/tenants/rh-trex-tenant/release-plan.yaml]: Great catch - should be patched!

**Review by scoheb — CHANGES_REQUESTED** [review]
looks great!

one item...I would encourage you to start with a `redhat` policy backed `IntegrationTestScenario` such as https://github.com/redhat-appstudio/tenants-config/blob/main/cluster/stone-prd-rh01/tenants/rhtap-spi-tenant/integration_test_scenario.yaml

**Review by scoheb — CHANGES_REQUESTED** [review]
name change needed

File: cluster/stone-prd-rh01/tenants/rh-trex-tenant/integration_test_scenario.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+      - name: revision
+        value: main
+      - name: pathInRepo
+        value: pipelines/enterprise-contract-redhat-no-hermetic.yaml
+    resolver: git
+---
+apiVersion: appstudio.redhat.com/v1beta1
+kind: IntegrationTestScenario
+metadata:
+  name: spi-enterprise-contract
```
[the remark this answers is not in the document]
Comment (gurnben) [reply cluster/stone-prd-rh01/tenants/rh-trex-tenant/integration_test_scenario.yaml]: Oops!  Fixed!

File: cluster/stone-prd-rh01/tenants/rh-trex-tenant/integration_test_scenario.yaml [hunk 3]
```diff
@@ -0,0 +1,39 @@
+apiVersion: appstudio.redhat.com/v1beta1
+kind: IntegrationTestScenario
+metadata:
+  name: remotesecret-enterprise-contract
+  namespace: rh-trex-tenant
+spec:
+  application: remotesecret
```
Comment (scoheb) [inline cluster/stone-prd-rh01/tenants/rh-trex-tenant/integration_test_scenario.yaml:7]: I think you are including an ITS for another tenant's application?

File: cluster/stone-prd-rh01/tenants/rh-trex-tenant/integration_test_scenario.yaml [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+        value: pipelines/enterprise-contract-redhat-no-hermetic.yaml
+    resolver: git
+---
+apiVersion: appstudio.redhat.com/v1beta1
+kind: IntegrationTestScenario
+metadata:
+  name: rh-trex-tenant-enterprise-contract
+  namespace: rh-trex-tenant
+spec:
+  application: spi
```
Comment (scoheb) [inline cluster/stone-prd-rh01/tenants/rh-trex-tenant/integration_test_scenario.yaml:27]: wrong application

**Review by scoheb — CHANGES_REQUESTED** [review]
2 more issues

File: cluster/stone-prd-rh01/tenants/rh-trex-tenant/integration_test_scenario.yaml (hunk 3 above)
Comment (gurnben) [reply cluster/stone-prd-rh01/tenants/rh-trex-tenant/integration_test_scenario.yaml]: Removing!

**Review by scoheb — APPROVED** [review]

**Review by ernesgonzalez33 — APPROVED** [review]

**Pull request merged**